modules/auxiliary/scanner/http/fortimail_login_bypass_detection.rb

```diff
@@ -0,0 +1,86 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+
+  def initialize
+    super(
+      'Name' => 'FortiMail Unauthenticated Login Bypass Scanner',
+      'Description' => %q{
+        This module attempts to detect instances of FortiMail vulnerable
+        against an unauthenicated login bypass (CVE-2020-9294).
+      },
+      'Author'         => [
+        'Mike Connor', # Initial Vulnerability discovery
+        'Juerg Schweingruber <juerg.schweingruber@redguard.ch>', # Vulnerability Re-Discovery
+        'Patrick Schmid <patrick.schmid@redguard.ch>' # Exploit Development & MSF module
+      ],
+      'References' =>
+        [
+          ['CVE',   '2020-9294'],
+          ['URL',   'https://fortiguard.com/psirt/FG-IR-20-045'],
+          ['URL',   'https://www.redguard.ch/blog/2020/07/02/fortimail-unauthenticated-login-bypass/']
```

Comment:
This link seems to be 404'd? And I can't find it on their blog? Do you know if there is an archived version available or check the link?

Reply:
Well, since the date in the URL seems to be 7/02/2020, my guess is that someone accidentally published it a week ahead of time.

Reply:
That’s correct, the blog post covering the exploit will be automatically published on July 2nd. But as the blog post only covers exploitation and the module covers only detection this is not directly relevant for the module.

Reply:
Ah, neat!